index.html

```diff
@@ -43,5 +45,195 @@ <h1>Hey There. I'm Carlyle</h1>
+            </div>
+            <div class="cardagraph">
+              <p class="cp-text">A daily selection of privately personalized reads; no accounts or sign-ups required. Due to my wider area, I decided to add more dummy text of my own making so as to appreciate the design more.</p>
+            </div>
+            <div class="cardtags">
+              <ul class="tags">
+                <li class="tag htag cardtagbg"><a href="#">html</a></li>
+                <li class="tag bootag cardtagbg"><a href="#">bootstrap</a></li>
+                <li class="tag rubytag cardtagbg"><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <div class="fullwidth">
+              <button type="button" class="probtn1">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="card card2">
+          <div class="card-content">
+            <div class="cardtitle">
+              <p class="ct-text">Profesional Art Printing Data</p>
+            </div>
+            <div class="cardagraph">
+              <p class="cp-text">A daily selection of privately personalized reads; no accounts or sign-ups required. Due to my wider area, I decided to add more dummy text of my own making so as to appreciate the design more.</p>
+            </div>
+            <div class="cardtags">
+              <ul class="tags">
+                <li class="tag htag cardtagbg"><a href="#">html</a></li>
+                <li class="tag bootag cardtagbg"><a href="#">bootstrap</a></li>
+                <li class="tag rubytag cardtagbg"><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <div class="fullwidth">
+              <button type="button" class="probtn1">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="card card3">
+          <div class="card-content">
+            <div class="cardtitle">
+              <p class="ct-text">Profesional Art Printing Data</p>
+            </div>
+            <div class="cardagraph">
+              <p class="cp-text">A daily selection of privately personalized reads; no accounts or sign-ups required. Due to my wider area, I decided to add more dummy text of my own making so as to appreciate the design more.</p>
+            </div>
+            <div class="cardtags">
+              <ul class="tags">
+                <li class="tag htag cardtagbg"><a href="#">html</a></li>
+                <li class="tag bootag cardtagbg"><a href="#">bootstrap</a></li>
+                <li class="tag rubytag cardtagbg"><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <div class="fullwidth">
+              <button type="button" class="probtn1">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="card card4">
+          <div class="card-content">
+            <div class="cardtitle">
+              <p class="ct-text">Profesional Art Printing Data</p>
```

Comment:
- [ ] [Required] I think you should have made each project title a header element. (h2 - h6). This will enhance SEO and semantics. Do this in all instances of this in your project.